.github/workflows/cypress.yml

```diff
@@ -65,11 +65,11 @@ jobs:
       - name: Get branch name
         uses: nelonoel/branch-name@v1.0.1
       - name: Run Cypress docs tests
-        uses: cypress-io/github-action@v2
+        uses: cypress-io/github-action@v4
         with:
           browser: chrome
           config: baseUrl=http://maas.io/docs
-          spec: "cypress/integration/docs-links/**/*.spec.ts"
+          spec: "cypress/e2e/docs-links/**/*.spec.ts"
```

Comment:
Is the rename to differentiate between e2e tests and component tests?

Reply:
Yep. The rename was done automatically via the interactive upgrade tool.